File: E-TicketingBackend/E-TicketingBackend/Model/ScheduleDTO.cs
using E_TicketingBackend.enums;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
namespace E_TicketingBackend.Model
{
    //Train Schedule DTO class
    public class ScheduleDTO
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string _id { get; set; }
        public string startingTime { get; set; }
        public string arrivalTime { get; set; }
        public  TrainDTO train { get; set; }
        public string startPoint { get; set; }
        public string endPoint { get; set; }
        public long ticketPrice { get; set; }
        public int Status { get; set; }
    }
}

File: mobileClient/app/src/main/java/com/sliit/eticket/dto/ScheduleDto.java
package com.sliit.eticket.dto;

import com.sliit.eticket.enums.CommonStatus;

import java.io.Serializable;

/*
 *  ScheduleDto class
 * */
public class ScheduleDto implements Serializable {
    private String _id;
    private String startingTime;
    private String arrivalTime;
    private  TrainDto train;
    private String startPoint;
    private String endPoint;
    private long ticketPrice;
    private int status;

    public ScheduleDto() {

    }


    public String get_id() {
        return _id;
    }

    public void set_id(String _id) {
        this._id = _id;
    }

    public String getStartingTime() {
        return startingTime;
    }

    public void setStartingTime(String startingTime) {
        this.startingTime = startingTime;
    }

    public String getArrivalTime() {
        return arrivalTime;
    }

    public void setArrivalTime(String arrivalTime) {
        this.arrivalTime = arrivalTime;
    }

    public TrainDto getTrain() {
        return train;
    }

    public void setTrain(TrainDto train) {
        this.train = train;
    }

    public String getStartPoint() {
        return startPoint;
    }

    public void setStartPoint(String startPoint) {
        this.startPoint = startPoint;
    }

    public String getEndPoint() {
        return endPoint;
    }

    public void setEndPoint(String endPoint) {
        this.endPoint = endPoint;
    }

    public long getTicketPrice() {
        return ticketPrice;
    }

    public void setTicketPrice(long ticketPrice) {
        this.ticketPrice = ticketPrice;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }
}
